.github/workflows/ci.yml

```diff
@@ -26,3 +26,7 @@ jobs:
         run: yarn demo:setup
       - name: Integration Tests
         run: yarn test:integrationOnly
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@v1
```

Comment:
What is this action doing with the sarif results?

Reply:
It was trying to get the results outputted to the Code Scanning Alerts under the Security tab, but I was unsuccessful. The docs around this is limited as it's only release since the start of Oct. I will remove it for the time being.